api.planx.uk/rateLimit.ts

```diff
@@ -7,6 +7,19 @@ const apiLimiter = rateLimit({
   max: 250,
   standardHeaders: true,
   legacyHeaders: false,
+  skip: (req: Request, _res: Response) => {
+    // add a mechanism for skipping rate limit when load testing (on local or staging only)
+    if (process.env.APP_ENVIRONMENT == "production") return false;
```

Comment:
without this (or a more convoluted solution like hosting Locust workers on remote servers with different IPs) I can't exceed the rate limit of 250 reqs per IP per 10 minute period, which of course is not sufficient for load testing purposes!

Reply:
Only thing I'm not totally following here is why we need a request header _and_ env var? Setting both and checking their equality is just an extra layer of assurance that we're bypassing intentionally?

Reply:
Latest commit feels much clearer, and still safe/catious :+1: